performance/TraceModel.js

```diff
@@ -56,8 +57,219 @@ const PageLoadingMetrics = (filename) => {
 	return {actualDCL, actualFCP, actualLCP};
 };
 
+class coreWebVitals {
+	constructor () {}
+
+	static async attachCwvLib (page) {
+		await page.evaluateHandle(() => {
+			// Including the coreWebVitals library
+			window.webVitals = (function (e) {
+				"use strict"; var n, t, r, i, o, a = -1, c = function (e) {
```

Comment:
Also, where does this minified code come from?

Reply:
I followed this guide: https://calendar.perfplanet.com/2023/inp-meets-puppeteer/

Reply:
I see. But is there any reason to do this?

there may be licensing issues if you copy and use the code as is.
How about changing it to use it by installing it from npm rather than retrieving it from CDN?

Reply:
I want to check one more thing. 
Why import it with cdn and not npm? We can do version control by importing it with npm. Currently, this code pulls the most recent version, the measurement code may change at any moment. When webVitals changes. I am concerned about dependencies being updated without our knowledge.